pages/login-notice/[type].tsx

```diff
@@ -18,6 +22,14 @@ export const NOTICE_CONTENT = {
     ],
     redirectUrl: notificationRoutes.manageNotifications,
   },
+  [SUBSCRIPTION_NOTIFICATIONS]: {
+    title: 'Sign up for updates',
+    content: [
+      'To sign up for updates, you need to sign in with GOV.UK One Login.',
+      'If you do not have a GOV.UK One Login, you can create one.',
+    ],
+    redirectUrl: `${notificationRoutes.manageNotifications}?action=${URL_ACTIONS.SUBSCRIBE}`,
```

Comment:
User-service state machine will need updated to accommodate this query string on the redirect. 

To avoid this: `/manage-notifications?action=subscribe?applyMigrationStatus=NOT_STARTED&findMigrationStatus=SUCCEEDED`

Reply:
Addressed in https://github.com/cabinetoffice/gap-user-service/pull/143